udata/features/territories/views.py

```diff
@@ -72,7 +72,7 @@ def compute_territory_dataset(territory, dataset, resource_id):
                      attachment_filename=attachment_filename)
 
 
-@blueprint.route('/territory/regions/', endpoint='home')
+@blueprint.route('/territoires/', endpoint='home')
```

Comment:
I'm a bit afraid of the complexity it adds. It opens the door to a whole new way to deal with URLs…